### DISCUSSION TITLE
欢迎来到 ”代码整洁的 JavaScript “ 的讨论区！

### DISCUSSION THREAD

**Discussion by beginor** [beginor/clean-code-javascript#2] [General]
## 👋 欢迎！

我们正在使用“讨论”作为与社区其他成员联系的场所。我们希望您：

  * 提出您想知道的问题。
  * 分享意见。
  * 与其他社区成员互动。
  * 欢迎他人，思想开放。请记住，这是我们一起建立的社区 💪 。

首先，请在下面对您的自我介绍进行评论，并告诉我们您对这个社区的看法。

**Comment by beginor** [comment]
发表意见时请保持代码整洁！

**Comment by ETTTTT** [comment]
‘使用 Object.assign 设置默认对象’ 这一条是不是可以用拓展运算符...代替呢

**Reply by beginor** [reply]
这个不一样， 拓展运算符通常用于原型连， 而 `Object.assign` 则是一次性为变量赋值多个属性值。
